include/constants/vars.h

```diff
@@ -113,7 +113,7 @@
 #define VAR_ROUTE101_STATE                   0x4060
 #define VAR_UNUSED_0x4061                    0x4061 // Unused Var
```

Comment:
Could probably interpolate what these unused vars could have been for

Reply:
Example: `VAR_ROUTE102_STATE` here

Reply:
Yup, they clearly allocated a block of vars for route/town states